README.md

```diff
@@ -3,6 +3,8 @@
 [![CRAN\_Status\_Badge](https://www.r-pkg.org/badges/version-last-release/pkgnet)](https://cran.r-project.org/package=pkgnet) [![CRAN\_Download\_Badge](https://cranlogs.r-pkg.org/badges/grand-total/pkgnet)](https://cran.r-project.org/package=pkgnet) [![Build Status](https://travis-ci.org/UptakeOpenSource/pkgnet.svg?branch=master)](https://travis-ci.org/UptakeOpenSource/pkgnet)
 [![codecov](https://codecov.io/gh/UptakeOpenSource/pkgnet/branch/master/graph/badge.svg)](https://codecov.io/gh/UptakeOpenSource/pkgnet)
 
+![](/docs/demo.gif)
```

Comment:
@bburns632 @jayqi I think we should IGNORE that for this PR because I have an idea for `pkgdown`.

I think we should remove `pkgdown` completely from the `master` branch. Instead, we should host off the `gh-pages` branch which @bburns632 manually pushes to. That would keep `master` clean, avoid "updated the pkgdown site" commits on the project, and make it more palatable to do custom stuff like writing a script to build the pkgdown site (doing things like fixing links).

Thoughts? Should I put this in an issue?

Reply:
@jameslamb that idea makes sense to me.

Reply:
@bburns632 @jameslamb @mattdzugan it looks like this is where to put the image so that it gets picked up by pkgdown.

`man/figures/`

https://github.com/r-lib/pkgdown/issues/280

Reply:
good find!
I can switch it up tomorrow!

Reply:
done!